EventsExpress/Validation/EventViewModelValidator.cs

```diff
@@ -1,20 +1,12 @@
 ﻿using System;
+using EventsExpress.Validation.Base;
 using EventsExpress.ViewModels;
 using FluentValidation;
 
 namespace EventsExpress.Validation
 {
-    public class EventViewModelValidator : AbstractValidator<EventViewModel>
+    public class EventViewModelValidator
+        : BaseEventViewModelValidator<EventViewModel>
```

Comment:
Do we really need this? I guess there are no endpoints that take this model as an argument